### DISCUSSION TITLE
✨ 2024 North America Field Day Selfie Thread ✨

### DISCUSSION THREAD

**Discussion by MetzinAround** [community/community#112578] [GitHub Education]
### Hi everyone,

Hello Field Day Attendees! 

This thread is for SELFIES 📸😊. IF you're attending our event in NYC today, this is your place to put the best shots of you and your new colleagues/friends. We're gonna pick a random discussion pair to receive a Mona Plushie! 

# RULES:

**Comment by Bricktech2000** [comment]
@Bricktech2000 and @aaron-ang
![Emilien and Aaron](https://github.com/community/community/assets/66650633/368b1bd6-5731-4c9a-8703-46b09575ae8c)

**Comment by ktsering2003** [comment]
![IMG_5011](https://github.com/community/community/assets/81478034/cf6cf807-1bbe-4082-827c-1fac25ae0e7c)


@ktsering2003
@dchen024
@SDRoan
@taisnguyen

**Comment by nolanlwin** [comment]
![image](https://github.com/community/community/assets/113712250/52c19498-057f-428f-acf5-6d3c6da1d607)

Nolan Lwin (i-am-nolan25) and Olaseni Gabriel (senigabriel)

We are from Pennsylvania and New Jersey.

**Comment by cchen002** [comment]
![IMG_3899](https://github.com/community/community/assets/104468123/90de5892-60e1-4e45-8ced-72198210bbc9)

Cher Chen - Rockville MD (cchen002)
Khannah Shaltiel - Forest Hills NY (khannah04)

**Comment by Vidushi-Gupta** [comment]
![image](https://github.com/community/community/assets/55969597/e96c465f-e8f3-461d-bfdc-f43e1215bd5b)
Y’all got the best photographer! Meet Javi from Miami (https://github.com/JaviTrek). I came in from Boston!

**Comment by DaniGall** [comment]
![IMG_0347](https://github.com/community/community/assets/144642043/6f4efa3f-5eef-42ce-bbf6-b3039ff732bc)
Dani - Queens College CUNY Girls who code 
Maggie - Hunter College CUNY Girls who code 
@maggeema

**Comment by TeresaVu** [comment]
leiassyun - Leia Yun - New York
TeresaVu - Teresa Vu - Gainesville, FL
![8C6229A7-2789-4520-8563-A752376C21C0](https://github.com/community/community/assets/95201928/e414efb4-95cb-41cf-9d8a-b4821b6ea928)

**Comment by markrubin920** [comment]
![65158740-6B3C-4576-8AD1-B1ED66448235](https://github.com/community/community/assets/109368910/d475e688-8fa9-4ea6-8827-612291961ed7)

@markrubin920 
@devinleamy
@LuneShine0

**Comment by bxique** [comment]
![IMG_0174](https://github.com/community/community/assets/65316481/9b2f2b8e-fe5a-44fa-88de-a83166aa0cc5)
Bernard X and Eddie Z.

**Comment by DevinLeamy** [comment]
![image](https://github.com/community/community/assets/45083086/9d7f4098-622b-4701-85ef-a3131a28144c)

@markrubin920 
@devinleamy
@luneshine0

**Comment by TeresaVu** [comment]
![Uploading A79C6961-665C-49B1-9DF5-966DFF2ABBBE.jpeg…]()
Maggie - maggeema - Brooklyn, NY
Teresa - TeresaVu - Gainesville, FL
![Uploading 72408D8C-C681-4E43-8415-1035CC2EE64B.jpeg…]()

**Comment by DaniGall** [comment]
![IMG_0352](https://github.com/community/community/assets/144642043/420e68c5-cbb7-4901-8667-f3e8c58d6ed8)
 @maggeema BK
@leiassyun
@TeresaVu

**Comment by leiassyun** [comment]
![image](https://github.com/community/community/assets/121903773/4d57bdf9-be0b-46da-9136-08f6527b6a2e)

@maggeema - Maggie Ma - Brooklyn, NY
@leiassyun - Leia Yun - New York, NY

**Comment by DevinLeamy** [comment]
![image](https://github.com/community/community/assets/45083086/2340001f-4271-4ea2-a2bd-0070744fb8b8)

@darren-yao
@Devinleamy

**Comment by DaniGall** [comment]
![IMG_0356](https://github.com/community/community/assets/144642043/02e80a5e-9df7-4008-8bd4-f50e61ead05c)
@juhwani
@bryanshabroski2

**Comment by lily-n20** [comment]
![IMG_2849](https://github.com/community/community/assets/113320013/05f8ebf0-6775-4233-8c9c-b734e923afd0)

zaria rosado / @zariarosa
lily nguyen / @lily-n20
jenny you / @muolii

**Comment by TeresaVu** [comment]
@mdtalhachy Talha Chowdhury Bennington, Vermont
@TeresaVu Teresa Vu from Gainesville, FL
![Uploading 5F152A56-34AD-44E3-84AF-01261217C8DA.jpeg…]()

**Comment by bxique** [comment]
![IMG_8994](https://github.com/community/community/assets/65316481/848331a5-65eb-4a13-9c65-4b93cd5d6626)
Bernard Xique
Eddie Zhang 
Bryan Luke Shabroski
Juhwan (Jay) Jung

**Comment by a-linco** [comment]
@a-linco & @prmuwanga

![IMG_3888](https://github.com/community/community/assets/105692753/0f61ec33-271b-4b69-837f-3b61f9af00c1)

**Comment by rthandra** [comment]
![IMG_8433](https://github.com/community/community/assets/109749791/226a37a3-6847-4917-9cd6-3b933272bdb2)

**Comment by efa5073** [comment]
![tempImageGITu5y](https://github.com/community/community/assets/58641822/174aabb1-30a8-4a2d-a86b-b158393eeef8)

**Comment by ellenchart** [comment]
![IMG_4752](https://github.com/community/community/assets/131402001/92e7bb19-865b-417e-a657-8f5c5f134516)

@ellenchart from Arlington, VA
@senigabriel from New Jersey

**Comment by kelly0921** [comment]
![IMG_3852](https://github.com/community/community/assets/46824496/e601d388-3e80-40f9-98d5-08fd9f2f4961)
@mdtalhachy
@kelly0921

**Comment by AlfGD** [comment]
@AlfGD and @MetzinAround 
From NYC and San Francisco
![IMG_8190](https://github.com/community/community/assets/125693641/b8a70581-cbda-4bf0-9ec2-ea1be148415e)

**Comment by khannah04** [comment]
![IMG_20240315_121007_988](https://github.com/community/community/assets/96746000/c77920ce-a66a-42e3-bddd-dd449b4b2946)


Adhishree and Khannah 
From Syracuse and NYC 
@adhishree123 @khannah04

**Comment by cchen002** [comment]
![278094A3-8C5B-43FF-83DE-94EAFF4C584B](https://github.com/community/community/assets/104468123/1688d46c-97f0-48dc-ae6d-75720546dac9)
Adhishree - Syracuse @adhishree123
Cher - Maryland @cchen002

**Comment by DaniGall** [comment]
![IMG_0383](https://github.com/community/community/assets/144642043/d17947c1-611a-4003-bf14-602e72497b82)

**Comment by kelly0921** [comment]
![IMG_3856](https://github.com/community/community/assets/46824496/3b04f83d-5bee-46aa-8b5d-f417b07f8e01)
@allisonchiangs
@kelly0921

**Comment by siannnhannn** [comment]
![IMG_0387](https://github.com/community/community/assets/117297888/345dc1e6-b706-4b31-830f-95d1c37c692d)

@siann23 @vaibhav-hariani

**Comment by khannah04** [comment]
![20240315_151028](https://github.com/community/community/assets/96746000/7b13bda5-f8c9-4bd9-9ba6-c15986f6ace2)

Cher, Lily, and Khannah 
Maryland, Rhode Island, NY

**Comment by ishaankalbhor** [comment]
![IMG_5739](https://github.com/community/community/assets/49913251/77b44f8b-038c-4c44-adee-9b7c0c4b7511)

**Comment by ishaankalbhor** [comment]
@heydc7
@ishaankalbhor 
![IMG_5739](https://github.com/community/community/assets/49913251/41f7898e-ae6e-446c-97f2-f58fcbcaf6e4)

**Comment by harry-data-g** [comment]
![PXL_20240315_190935361.jpg](https://github.com/community/community/assets/60471123/b97e374b-4df3-4a4d-9e4f-8831744f06b2)

I met him today and got to know about his amazing contributions

**Comment by harry-data-g** [comment]
![PXL_20240315_191144287.jpg](https://github.com/community/community/assets/60471123/07286e6b-9692-42fc-b6a4-28f1f8f402ea)

I met him today as his enthusiastic learning attitude actually influenced me pretty much

**Comment by stephanienguyen2020** [comment]
![IMG_5419](https://github.com/community/community/assets/62029350/a7885fe5-a653-4b89-ae32-3b9e14d7083a)

I met Joe and Edward today. Their energies were incredible 👌🏾

**Comment by taisnguyen** [comment]
![image](https://github.com/community/community/assets/26637127/a6f8ffec-3ac4-46c6-8758-f5b228ce145d)
@dchen024
@taisnguyen

**Comment by MetzinAround** [comment]
EXCELLENT selfie thread y'all!! thanks for coming to Field Day North America 2024!! 

Make sure you have the student developer pack!! https://education.github.com/pack
